src/NotionExporter.ts

```diff
@@ -68,22 +69,21 @@ export class NotionExporter {
   }
 
   private pollTask = (
-    taskId: string,
-    pollInterval: number = 50
+    taskId: string
   ): Promise<string> =>
     new Promise((resolve, reject) => {
       const poll = async () => {
         const task = await this.getTask(taskId)
         if (task.state === "success" && task.status.exportURL)
           resolve(task.status.exportURL)
         else if (task.state === "in_progress" || task.state === "not_started") {
-          setTimeout(poll, pollInterval)
+          setTimeout(poll, this.config.pollInterval || this._DEFAULT_POLL_INTERVAL)
```

Comment:
... that would change this to:
```suggestion
          setTimeout(poll, this.config.pollInterval || defaultConfig.pollInterval)
```
(same below)